Packs/MicrosoftTeams/Scripts/GrantAccessPolicy/GrantAccessPolicy.yml

```diff
@@ -0,0 +1,42 @@
+args:
```

Comment:
please add test playbook for this scriipt

Reply:
It's a problem because the user must put it's admin credentials and it's not something we can get from the integration input. besides, we need a special self-service to run it, like the one Tomer created and not just any self-service such as the nightly one.
I'll add to Centrify an example of how to run that script with our credentials.